db/sql/sql.go

```diff
@@ -1299,3 +1315,48 @@ func (db *DB) SetMaxOpenConns(maxIdleConns int) {
 func safeMysqlContext(_ context.Context) context.Context {
 	return context.Background()
 }
+
+type Like map[string]interface{}
+
+func (lk Like) ToSql() (sql string, args []interface{}, err error) {
+	var (
+		exprs []string
```

Comment:
`exprs` is either empty (in case of error) or has `len(lk)` elements, we can preallocate memory here.

Reply:
this just a copy of the function in squirrel..had to copy it here just for the ESCAPE part..intent of the function was to use as exactly as what squirrel library has and just inject ESCAPE for the queries..

Reply:
i guess Wojciech meant to change this line to `exprs = make([]string, 0, len(lk))` to avoid unnecessary allocation in for loop